src/Task_3.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Arrays;
+import java.util.Random;
+
+public class Task_3 {
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[][] massivForDiagonals = new int[5][5];
+        System.out.println("Массив: ");
+
+        for (int i = 0; i < massivForDiagonals.length; i++) {
+            for (int j = 0; j < massivForDiagonals[i].length; j++) {
+                massivForDiagonals[i][j] = random.nextInt(10);
+
+            }
+            System.out.println(Arrays.toString(massivForDiagonals[i]));
+
+        }
+        System.out.println("Перая диагональ: ");
+        for (int i = 0; i < massivForDiagonals.length; i++) {
+            int Diagonal;
+            Diagonal = massivForDiagonals[i][i];
+            System.out.print(Diagonal + " ");
+
+        }
+        System.out.println(" ");
+        System.out.println("Вторая диагональ: ");
+        for (int i = 0; i < massivForDiagonals.length; i++) {
+            for (int j = 0; j < massivForDiagonals.length; j++) {
+                if (j == massivForDiagonals.length - i - 1) {
+                    System.out.print(massivForDiagonals[i][j] + " ");
```

Comment:
Can we do that using a single cycle?

Reply:
I don’t know how yet, to maintain the ratio j=n-i-1 you need two cycles.
The only thing that comes to mind so far is to create a new variable and somehow replace the loop with it. If I don’t do it by Tuesday, then this will be a good question at the beginning of the lesson :)

Reply:
Done